src/aframe-streetmix-parsers.js

```diff
@@ -528,7 +528,7 @@ function createMicroMobilityElement (variantList, positionX, segmentType, length
   }
   const bikeLength = 2.03;
   const positionZ = randomPosition(reusableObjectEl, 'z', length, bikeLength);
-  reusableObjectEl.setAttribute('position', positionX + ' 0 ' + positionZ);
+  reusableObjectEl.setAttribute('position', positionX + ' 0.2 ' + positionZ);
```

Comment:
Confused a bit about this, hard coding the Y position at 0.2 means that the cyclist will float if not on the sidewalk or higher elevation surface?